jobdispatcher/build.gradle

```diff
@@ -22,10 +22,6 @@ android {
         debug {
             testCoverageEnabled true
         }
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt')
-        }
```

Comment:
Why does removing this fix the issue? Is your app obfuscating the PendingCallback? What are your proguard settings?